### ISSUE TITLE
Check-in read view for scrum master of all students in attendance

### ISSUE THREAD

**Issue by HA-Kodes** [CodersCampus/cp#648]
This is a great collection of insights and information but the task itself is to go through this and peel out meaningful tickets.

The purpose of this view is to show which students are ready & willing for live-coding

Also see #679 for one key thing already peeled out

![image](https://github.com/user-attachments/assets/d7750339-ca09-448a-90a4-d7046bd65116)

purpose: to organize Check-in meeting for Coder and Guide based on readiness
this is a tabular / spreadsheet type view

- [ ] add Student Name (Name)
- [ ] data showing date filter by those in attendance (Date)
- [ ] keep Next Assignment view (Assignment)
- [ ] keep This is a Live Coding Session, filter
- [ ] keep I am available / willing to Code (Ready)
- [ ] remove Blocker checkbox
- [ ] remove Blocker Description

in other words, just need a tabular view for Name, Assignment, and Ready
